lib/ProductOpener/Ingredients.pm

```diff
@@ -1244,13 +1244,22 @@ sub parse_ingredients_text($) {
 								my $regexp = $ingredient_processing_regexp_ref->[1];
 								if (
 									# English, French etc. match before or after the ingredient, require a space
-									(($product_lc =~ /^(en|es|it|fr)$/) and ($new_ingredient =~ /(^($regexp)\b|\b($regexp)$)/i))
+									(
+										#($product_lc =~ /^(en|es|it|fr)$/)
+										( ($product_lc eq 'en') or ($product_lc eq 'es') or ($product_lc eq 'fr') or ($product_lc eq 'it') )
+										and ($new_ingredient =~ /(^($regexp)\b|\b($regexp)$)/i)
+									)
+									
 									#  match after, do not require a space
 									# currently no language
-									or  (($product_lc =~ /^(xx)$/) and ($new_ingredient =~ /($regexp)$/i))
+									or ( ($product_lc eq 'xx') and ($new_ingredient =~ /($regexp)$/i) )
```

Comment:
In fact that line can be commented out (there's no language "xx", it's just there in case for some languages we need to match after, but currently we don't do it).

Reply:
```suggestion
									#or ( ($product_lc eq 'xx') and ($new_ingredient =~ /($regexp)$/i) )
```